### ISSUE TITLE
Essential feature missing! Drag-n-drop unpacking

### ISSUE THREAD

**Issue by tu5** [mate-desktop/engrampa#177]
Please add an option to unpack files or folders by drag-n-dropping them. Thanks.

**Comment by sc0w** [comment]
I can extract files/folders with drag and drop from engrampa to caja with no problems

**Comment by tu5** [comment]
Yeah right, it's work with Caja but seem to not with third party software like Chrome or Calibre as I tested

**Comment by raveit65** [comment]
Isn't Chrome a webbrowser?
And calibre is a ebook application.
I am in doubt that we should really support any other application than a filebrowser.
Does file-roller support those apps?

**Comment by tu5** [comment]
Yes, they both support drag-n-drop from Caja.


On Tue, May 9, 2017, at 08:56, raveit65 wrote:
> Isn't Chrome a webbrowser?
>  And calibre is a ebook application.
>  I am in doubt that we should really support any other application
>  than a filebrowser.>  Does file-roller support those apps?


> — You are receiving this because you authored the thread. Reply to
> this email directly, view it on GitHub[1], or mute the thread[2].> 


Links:

  1. https://github.com/mate-desktop/engrampa/issues/177#issuecomment-300077910

**Comment by raveit65** [comment]
I ask you if file-roller support drag&drop to those applications?
Please be carefull with posting stuff like this here.
```
#177
```
This can make links to other reports and we can't remove it.

**Comment by tu5** [comment]
Seems to not support it. Looks like it's just a linux thing, on windows, it works anywhere.

**Comment by sc0w** [comment]
good feature, for example to drag n drop to audacious or another audio player, and it will open from the temp folder

file-roller and xarchiver can't do that

**Comment by meequz** [comment]
Would be awesome feature. However, is it possible to implement at all? The file need to be extracted first, it doesn't exist on disk at the drag moment. Even if start to extract it immediately when user hold it, extracting may take much time.

**Comment by haarp** [comment]
Extracting via drag'n'drop to a file manager such as Thunar works. Doing the same into another program does **not**. One has to first DND into a Thunar location, and from there DND into another app. Annoying and unnecessarily complex.

**Comment by ajsnyde** [comment]
Drag-n-drop into applications is a valuable feature. 7-zip on Windows can drag-n-drop subtitles files into VLC, but for my linux desktop I've got to drag-n-drop it into the desktop, then drag-n-drop it from the desktop into VLC. Makes me look stupid on movie night 🍿
